pkg/server/image_status.go

```diff
@@ -17,14 +17,45 @@ limitations under the License.
 package server
 
 import (
-	"errors"
-
 	"golang.org/x/net/context"
 
 	"k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
-// ImageStatus returns the status of the image, returns nil if the image doesn't present.
+// ImageStatus returns the status of the image, returns nil if the image isn't present.
 func (c *criContainerdService) ImageStatus(ctx context.Context, r *runtime.ImageStatusRequest) (*runtime.ImageStatusResponse, error) {
-	return nil, errors.New("not implemented")
+	ref := r.GetImage().GetImage()
+	// if the passed ref is a digest.. and is stored the following get should work
```

Comment:
If it is a image reference, we could:
1) Get the id->tags, and id->digest mapping from our checkpoint;
2) Get other information from containerd image/content store, so that we can make sure the image does exist in containerd.

If it is a image id (digest), we should get a image reference from our checkpoint, and then do the same with above.